src/Middleware/HttpOverrides/src/PublicAPI.Unshipped.txt

```diff
@@ -12,3 +12,11 @@ static Microsoft.AspNetCore.HttpOverrides.ForwardedHeadersDefaults.XForwardedPre
 static Microsoft.AspNetCore.HttpOverrides.ForwardedHeadersDefaults.XOriginalPrefixHeaderName.get -> string!
 static Microsoft.AspNetCore.HttpOverrides.IPNetwork.Parse(System.ReadOnlySpan<char> networkSpan) -> Microsoft.AspNetCore.HttpOverrides.IPNetwork!
 static Microsoft.AspNetCore.HttpOverrides.IPNetwork.TryParse(System.ReadOnlySpan<char> networkSpan, out Microsoft.AspNetCore.HttpOverrides.IPNetwork? network) -> bool
+Microsoft.AspNetCore.Builder.ForwardedHeadersOptions.ForwardedPortHeaderName.get -> string!
+Microsoft.AspNetCore.Builder.ForwardedHeadersOptions.ForwardedPortHeaderName.set -> void
+Microsoft.AspNetCore.Builder.ForwardedHeadersOptions.OriginalPortHeaderName.get -> string!
+Microsoft.AspNetCore.Builder.ForwardedHeadersOptions.OriginalPortHeaderName.set -> void
+Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.XForwardedPort = 16 -> Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders
+static Microsoft.AspNetCore.HttpOverrides.ForwardedHeadersDefaults.XForwardedPortHeaderName.get -> string!
+static Microsoft.AspNetCore.HttpOverrides.ForwardedHeadersDefaults.XOriginalPortHeaderName.get -> string!
+Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.All = Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.XForwardedFor | Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.XForwardedHost | Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.XForwardedProto | Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.XForwardedPrefix | Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders.XForwardedPort -> Microsoft.AspNetCore.HttpOverrides.ForwardedHeaders
```

Comment:
My bad: I forgot to say that you'll need to remove the old new version of `All` on line 9 (but keep the `*REMOVED*` one).